snapcraft/hooks/connect-plug-graphics-core22

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -eu
```

Comment:
Please add a comment near the top of this file explaining what it does for future readers.